### PR TITLE
fix:  wrong module name in maven multi-level projects #677

### PR DESCRIPTION
**Pull request by GiuseppeMP**
Make sure these boxes are checked before submitting your pull request:

- [x] Add fixtures and spec when implementing or updating a test runner
- [] Update the README accordingly (no change needed)
- [] Update the Vim documentation in `doc/test.txt` (no change needed)

### CODE REVIEW

File: autoload/test/java/maventest.vim [hunk 1]
```diff
@@ -67,10 +67,10 @@ endfunction
 
 function! s:get_maven_module(filepath)
   let project_dir = s:GetJavaProjectDirectory(a:filepath)
-  let l:module_name = fnamemodify(project_dir, ':t')
+  " let l:module_name = fnamemodify(project_dir, ':t')
```
Comment (codeinabox) [inline autoload/test/java/maventest.vim:70]: If this is no longer needed, could it please be removed

**Review by codeinabox — CHANGES_REQUESTED** [review]
Thank you for this, looking good! Could you please remove the comment and also the `.settings` directory in the fixtures

**Comment by GiuseppeMP** [comment]
@codeinabox Hey,

Comments resolved.

**Review by codeinabox — APPROVED** [review]
Thank you for this!

**Pull request merged**